backend/test/controllers/bucket_test.ts

```diff
@@ -91,5 +93,31 @@ import {
     )
   }
 
+  @test async 'can be returned as page repositories'() {
+    const new_user = createNewUser(this.testUserData);
+    const new_bucket = createNewBucket(this.testBucketData(new_user._id));
+
+    await saveUser(new_user).then(saved_user =>
```

Comment:
This shows how you could use a timeout with await 
https://stackoverflow.com/questions/33289726/combination-of-async-function-await-settimeout